### ISSUE TITLE
Bug: Recipient address can be changed if you refresh an in-progress transfer

### ISSUE THREAD

**Issue by aryabhuptani** [connext/vector-modal#49]
Flow:
1. Start a transfer to recipient 0x1
2. Refresh after the deposit but before transfer initiates
3. Plug in a new recipient 0x2 and reopen the widget.

I haven't tried to repro this myself, but it was reported by quant.finance who have been testing our widget.

Ideal UX here is that once a user has provided a recipient for a given transfer, completing that in-progress transfer *should* go to the same recipient instead of taking a new recipient.

**Comment by sanchaymittal** [comment]
@ArjunBhuptani Not sure if this is a bug.
we can see this as a feature too. An instance would be if someone wants to changes the receiver address after starting the transfer. They get to have a freedom of doing so.

**Comment by LayneHaber** [comment]
Yeah -- I think the better solution here would be to have the resumed transfers be user-initiated. Something like "we detected an existing transfer, would you like to complete/recover/or add to it? Do you still want it to go to xxxx" instead of just restarting

**Comment by sanchaymittal** [comment]
Yeah, make sense. I discussed this with @rhlsthrm too.
We detect balance at senderChain channel, show [5-10 Sec window] screen 

desc: Found existing channel balance for __ Token at Sender Chain

[Complete this pending Transfer], [Increase Transfer Amount] Or [Recover funds].

This will need some designs improv, also some Higher level docs for user to understand about channel balance.
Also some field at swapScreen showing existing channel balance.

cc: @LayneHaber @rhlsthrm
